hacspec/src/bigint_integers.rs

```diff
@@ -0,0 +1,158 @@
+#![allow(unused_variables)]
+use crate::prelude::*;
+
```

Comment:
I don't understand this file, why would we need to use directly `BigInt` while we have `abstract_integers` that implement what we want ? Right now `abstract_integers` is implemented using `BigInt` but eventually we can swap it with a constant-time implementation (maybe from HACL*).

Reply:
see previous comments